src/index.html

```diff
@@ -22,6 +22,389 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="./index.html"
+        class="logo-link"
+      >
+        <img
+          src="./images/logo.png"
+          alt="logo"
```

Comment:
For logos, use the format: "[Organization name] logo." 

![image](https://github.com/mate-academy/layout_catalog/assets/54860641/4085cbce-cce1-46d8-a54e-dadc7df94fdf)

Reply:
Thanks!